static-site/app/groups/[[...groups]]/group-listing-page.tsx

```diff
@@ -180,17 +181,29 @@ async function _resourcesCallback(): Promise<Group[]> {
     return {name, groupName, nameParts, sortingName, url};
   }).filter((r): r is Resource => r!==undefined);
 
-  const groups: Group[] = Array.from(new Set(resources.map((r) => r.groupName)))
-    .map((groupName) => {
+  const groups: Group[] = await Promise.all(Array.from(new Set(resources.map((r) => r.groupName)))
```

Comment:
This isn't great because rendering the `list-resources` component is now blocked on fetching all group logos. There should be a way to block only the logo part – I'll look into this.